### DISCUSSION TITLE
Python or JS for beginner

### DISCUSSION THREAD

**Discussion by AM-SEAS** [community/community#63351] [Actions]
### Select Topic Area

Question

### Body

?

**Comment by ArnM867** [comment]
Python

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
